api/trace/testtrace/generator.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testtrace
+
+import (
+	"encoding/binary"
+
+	"go.opentelemetry.io/otel/api/core"
+)
+
+type Generator interface {
```

Comment:
Please add comments to the to the exported types and methods that are not implementing the OTel API. Readers will appreciate a brief explanation for how the test-specific stuff is used, and for the mock maybe a brief mention of how they fit into a test.